src/App/Options/Cmd.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE DeriveAnyClass   #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TemplateHaskell  #-}
+
+module App.Options.Cmd where
+
+import App.Options.Cmd.Help       as C
+import App.Options.Cmd.KafkaToSqs as C
+import App.Options.Cmd.SqsToKafka as C
+import Control.Lens
+import Data.Monoid
+import Options.Applicative
+
+data Cmd
+  = CmdOfCmdHelp        { _cmdHelp        :: CmdHelp        }
+  | CmdOfCmdKafkaToSqs  { _cmdKafkaToSqs  :: CmdKafkaToSqs  }
```

Comment:
It is usually a bad practice to add record fields to ADT constructors because they become partial functions

Reply:
Doesn't `Control.Lens` generate prisms for these?

Reply:
Prisms are generated for constructors AFAIK.
Like, with `makePrism ''Cmd` you'll have a prism `_CmdOfCmdKafkaToSqs` because you have constructor `CmdOfCmdKafkaToSqs`. But record members are still just functions.